homeassistant/components/epson/config_flow.py

```diff
@@ -1,21 +1,21 @@
 """Config flow for epson integration."""
+import logging
+
 import voluptuous as vol
 
 from homeassistant import config_entries
-from homeassistant.const import CONF_HOST, CONF_NAME, CONF_PORT
+from homeassistant.const import CONF_HOST, CONF_NAME
 
 from . import validate_projector
 from .const import DOMAIN
-from .exceptions import CannotConnect
+from .exceptions import CannotConnect, PoweredOff
 
 DATA_SCHEMA = vol.Schema(
-    {
-        vol.Required(CONF_HOST): str,
-        vol.Required(CONF_NAME, default=DOMAIN): str,
-        vol.Required(CONF_PORT, default=80): int,
```

Comment:
I suggest to keep the port as an optional parameter.

Reply:
if pure TCP connection will be accepted there is no way to switch ports.